frontend/src/api/actions/init.js

```diff
@@ -0,0 +1,150 @@
+import {
+    setActiveSession,
+    // setActiveUserRole,
+    fetchActiveUser,
+    fetchSessions,
+    fetchApplicants,
+    fetchApplications,
+    fetchAssignments,
+    fetchContractTemplates,
+    fetchInstructors,
+    fetchPositions
+} from "./";
+import { setGlobals, globalsSelector } from "../../views/globals/actions";
```

Comment:
I don't like the idea of importing from views into `api/`. `api/` should be self-contained.